### DISCUSSION TITLE
Thumbnails for files in repository

### DISCUSSION THREAD

**Discussion by tsadarsh** [community/community#38912] [Repositories]
### Select Topic Area

General

### Body

The current GitHub repository files look like this:

![Screenshot from 2022-11-12 12-33-21](https://user-images.githubusercontent.com/66778010/201461974-cbf81a68-52e5-4d0f-bb6c-65bdcaa29f8b.png)

Isn'it it redundant (and boring) to have the same SVG for all types of files. **So I decided to add thumbnails for each type of file**. This is how it looks now:

![Screenshot from 2022-11-12 12-36-08](https://user-images.githubusercontent.com/66778010/201462060-4d2e1aec-b98d-4f0c-b923-2dc2a2af8e3f.png)

I am curious to know if this type of design was thought of before. Can this be implemented? I am willing to work more on this.

**Comment by jfullstackdev** [comment]
yup, it's the online Editor, just replace the URL from `.com` to `.dev`, or when you are in the repository, just simply hit `.` and it will redirect

it's actually VS Code look-and-feel, just like this 

![image](https://user-images.githubusercontent.com/47092464/201530121-5f7b2aa8-a483-4c3e-b39e-ea3d757bb057.png)
